### ISSUE TITLE
Losses

### ISSUE THREAD

**Issue by yuntaodu** [stevebottos/owl-vit-object-detection#13]
Hi, Great job. I have a question, what does this use for in losses.py of line 37-38

 pos_loss = (torch.pow(1 - torch.exp(-pos_loss), 2) * pos_loss).sum(dim=1).mean()
 neg_loss = (torch.pow(1 - torch.exp(-neg_loss), 2) * neg_loss).sum(dim=1).mean()

**Comment by sharad5** [comment]
I think this is implementing [Focal Sigmoid Loss](https://arxiv.org/pdf/1708.02002.pdf). 
The pos_loss and neg_loss in this expression are log(p_t) and log(1-p_t) and gamma = 2.
